modular_zubbers/code/modules/languages/empathy.dm

```diff
@@ -46,16 +46,16 @@
 	if(speech_args[SPEECH_LANGUAGE] in languages_native) // Speaking a native language?
 		return modify_speech(source, speech_args)
 
-
-
 /obj/item/organ/internal/tongue/shadekin/modify_speech(datum/source, list/speech_args)
 	ASYNC
 		actually_modify_speech(source, speech_args)
 	speech_args[SPEECH_MESSAGE] = "" // Makes it not send to chat verbally.
 
-/obj/item/organ/internal/tongue/shadekin/proc/actually_modify_speech(datum/source, list/speech_args)
+/obj/item/organ/internal/tongue/shadekin/proc/actually_modify_speech(datum/source, list/speech_args, user)
```

Comment:
yes this one is wrong but I'm not amending micro-commits at 0300 when there's an actual issue. Like how the do_after at line 61 isn't firing for implant users. I thiiiink the source isn't carrying over?